README.md

```diff
@@ -16,7 +16,7 @@ Run:
 
 ```
 npm install
-npx convex dev
+npm run dev
```

Comment:
Hmm even with the improvements, it would be perhaps better if people read the part about env vars first...

Reply:
The problem is there isn't anywhere to put them until you start to run it. 
Ideally it could be like `npx convex setenv STRIPE_WEBHOOKS_SECRET <value>` - maybe that could lazily create a project if it's the first thing run?
Maybe we need `npx convex init` after all?
I don't like telling people to copy a variable if they don't have a place to paste it yet.